mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -144,6 +144,24 @@ def annotate_builtin_endpoints(self, name, namespace='default'):
                     "Retry updating annotating endpoints {}".format(name))
                 get_body = True
 
+    def annotate_builtin_pods(self, name, namespace='default'):
+        get_body = True
+        while get_body:
```

Comment:
(namespace, name) could be duplicated among multiple tenants for Arktos.

Reply:
This annotation is used to inform arktos that the pod creation is ready. Do we still need this annotation tho? Hongwei mentioned that it's good to have field. We can track the pod creation status by "STATUS" field in pod description. 

https://github.com/clu2xlu/mizar/blob/arktos-dns-service-deploy/mizar/dp/mizar/workflows/builtins/pods/create.py#L100-L101

Reply:
This annotation is added after the pod is set to be Provisioned, which does not work in two clusters env. Even if we want add this annotation, we should do it in network controller, right?